app/rtc_s1_parallel.py

```diff
@@ -13,6 +13,1169 @@
 
 from rtc.runconfig import RunConfig
 
+import logging
+import numpy as np
+from osgeo import gdal
+
+import isce3
+
+from s1reader.s1_burst_slc import Sentinel1BurstSlc
+
+from rtc.geogrid import snap_coord
+from rtc.runconfig import RunConfig
+from rtc.mosaic_geobursts import compute_weighted_mosaic_raster, compute_weighted_mosaic_raster_single_band
+from rtc.core import create_logger, save_as_cog
+from rtc.h5_prep import save_hdf5_file, create_hdf5_file, BASE_HDF5_DATASET
+
+logger = logging.getLogger('rtc_s1')
+
+
+def _update_mosaic_boundaries(mosaic_geogrid_dict, geogrid):
```

Comment:
Did you change this function from `rtc_s1.py`? If not, you can import it from that module. You just need to remove the leading underscore `_` from the function name. Same for the other functions below that can just be imported from `rtc_s1.py`.